### PR TITLE
Documentation build for new ci

### PR DESCRIPTION
**Pull request by enys**

### CODE REVIEW

**Comment by enys** [comment]
@davidrapin we should be good to go with this one

File: package.json [hunk 1]
```diff
@@ -24,12 +28,17 @@
...
+    "bump:patch": "bump2version patch && npm version --no-git-tag-version patch ",
+    "bump:minor": "bump2version minor && npm version --no-git-tag-version minor",
+    "bump:major": "bump2version major && npm version --no-git-tag-version major"
   },
   "dependencies": {},
   "devDependencies": {
     "dokapi": "1.0.26",
-    "fs-extra": "2.0.0"
+    "fs-extra": "2.0.0",
+    "linkurious-server": "lk-develop"
```
Comment (davidrapin) [inline package.json:42]: why the "lk-" prefix?

File: package.json [hunk 2]
```diff
@@ -1,8 +1,12 @@
 {
   "name": "documentation",
   "description": "Linkurious Enterprise documentation",
-  "version": "2.9.5",
+  "version": "2.10.2",
```
Comment (davidrapin) [inline package.json:4]: how is the "latest" version of the doc defined?

**Review by davidrapin — COMMENTED** [review]
Could you detail which features are covered by this PR?
- support for "latest" shortcut read from a json file
- dynamically generated changelog from jira
- doc preprod working for to-be-released LKE versions
- other?

File: package.json (hunk 1 above)
Comment (enys) [reply package.json:42]: to avoid conflicting with public tags

you can do things like npm install --tag lk-develop.

without the prefix we could pull strange tags from public npm registry

File: package.json (hunk 2 above)
Comment (enys) [reply package.json:4]: Not sure of your question.

If the question is : how is the link set to the latest on the doc website: on asset download, check the prerelease flag, and if not prerelease, then set as latest. THis is done by documentation-root repo.

**Comment by enys** [comment]
This is combined with work on documentation-root repository.
all three should be covered. Should: I need another round of testing, with the asset built by the merge of this PR.

Other : Sort order of versions

> Could you detail which features are covered by this PR?
> 
> * support for "latest" shortcut read from a json file
> * dynamically generated changelog from jira
> * doc preprod working for to-be-released LKE versions
> * other?

File: package.json (hunk 2 above)
Comment (davidrapin) [reply package.json:4]: Yes that was my question, thank you

File: package.json (hunk 1 above)
Comment (davidrapin) [reply package.json:42]: I don't understand your answer :/
I'm not sure what you mean by "public tags" here, and what kind of conflict you are refering to.
I'm also missing context to understand the last phrase.

Comment (enys) [reply package.json:42]: Back from the beginning:
What is lk-develop -> it is a dist-tag ( see https://docs.npmjs.com/cli/v6/commands/npm-dist-tag).
It is produced by the ci: lk-{BRANCH_NAME}

We are not allown using dist-tags. Public projects (may) use them also. And we don't want to have similar ones.

Comment (davidrapin) [reply package.json:42]: after discussing, I doubt we need the "lk-" prefix

Comment (enys) [reply package.json:42]: Lets try to sum it up for posterity : 
The case where it would make sense : using a project level 'npm install --tag lk-develop' and there is no immediate plan for this.
Another case would be for a grep/sed like automation process.
@davidrapin could you list the cons of the prefix?
I still have a bad (and unexplained feeling) about dropping the prefix.

Comment (enys) [reply package.json:42]: Most Popular public repositories have a 'next' tag for example. (Not knowing if that is a pro or con argument for prefix)
